frontend/components/panel/js/panel.component.ts

```diff
@@ -138,7 +138,7 @@ class PanelController implements ng.IController {
 		*/
 		angular.element(document).bind("mouseup", () => {
 			this.activate = true;
-			this.$scope.$apply();
+			//this.$scope.$apply();
```

Comment:
remove unless there is a reason why it needs to stay (in which case we need a comment here)